src/asv_spyglass/_asv_ro.py

```diff
@@ -14,19 +12,21 @@ class ReadOnlyASVBenchmarks:
 
     api_version = 2
 
-    def __init__(self, benchmarks_file: Path, regex: Union[str, list[str]] = None):
+    def __init__(self, benchmarks_file: Path, regex: str | list[str] = None):
```

Comment:
Missing `None` in the type

```suggestion
    def __init__(self, benchmarks_file: Path, regex: str | list[str] | None = None):
```